Assets/Scripts/Models/TileType.cs

```diff
@@ -207,104 +172,64 @@ public override int GetHashCode()
     {
         return Type != null ? Type.GetHashCode() : 0;
     }
-    
-    // Overrides ToString() to be able to do localizaton in character
+
     public override string ToString()
     {
         return Type;
     }
 
-    #region IXmlSerializable implementation 
-
-    public XmlSchema GetSchema()
+    public System.Xml.Schema.XmlSchema GetSchema()
     {
         return null;
     }
 
-    public void ReadXml(XmlReader reader_parent)
+    public void ReadXml(XmlReader parentReader)
     {
-        Type = reader_parent.GetAttribute("tileType");
-
-        XmlReader reader = reader_parent.ReadSubtree();
+        Type = parentReader.GetAttribute("tileType");
 
+        XmlReader reader = parentReader.ReadSubtree();
         while (reader.Read())
         {
             switch (reader.Name)
             {
                 case "Name":
-                    reader.Read();
-                    Name = reader.ReadContentAsString();
-                    break;
+                reader.Read();
```

Comment:
I thought that we were doing swicthes like:

```
switch (reader.Name)
{
    case "Name":
        // Code goes here
        break;
    case "Description":
        // ...
        break;
}
```

It was already changed in furniture from a different style.